src/pomodoro.sh

```diff
@@ -250,9 +250,8 @@ function calculate_missing_time()
     m)
       time_value=$((60 * "$time_value"))
       ;;
-    s)
-      time_value="$time_value"
-      ;;
+    s) ;;
+
```

Comment:
Maybe the `s` case could be totally left out. But maybe this way is more readable.

Reply:
Yeah... let's remove this `s)`